portfolio/src/main/webapp/script.js

```diff
@@ -42,6 +42,28 @@ function addToDOM(textResponse) {
   textContainer.innerText = textResponse;
 }
 
+function loadHomePageContent() {
+  fetchUserLoginStatus();
+  getComments();
+}
+
+function fetchUserLoginStatus() {
+  const commentsFormMessageContainer = document.getElementById("comments-form-message");
+  fetch('/login').then(response => response.json()).then((result) => {
+    if (result.status) {
+      commentsFormMessageContainer.innerText = '';
+      commentsFormMessageContainer.appendChild(createSpanElement("Click "));
```

Comment:
nit: if you aren't going to apply a class to these spans in order to style them, you can just use document.createTextNode() to avoid unnecessary elements

Reply:
Got it!